From eddd8747ff46d033c10abf28a4ee54b2256843d0 Mon Sep 17 00:00:00 2001
From: Sean Mackrory <sean@cloudera.com>
Date: Mon, 13 Jan 2014 11:42:27 -0800
Subject: [PATCH 171/596] CLOUDERA-BUILD. Harmonizing commons-logging version
 to 1.1.3

(cherry picked from commit 0df9cab9e4648ec734ece60c7498cf1be43b585f)
(cherry picked from commit 2461cc419a9a696572c284c8555923f1f49201fe)
(cherry picked from commit b66182379bf81ae3eccf40b1c85b85b6445575b5)
---
 hadoop-mapreduce1-project/ivy/libraries.properties |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-mapreduce1-project/ivy/libraries.properties b/hadoop-mapreduce1-project/ivy/libraries.properties
index eb9f9fb..ef07e22 100644
--- a/hadoop-mapreduce1-project/ivy/libraries.properties
+++ b/hadoop-mapreduce1-project/ivy/libraries.properties
@@ -29,7 +29,7 @@ commons-codec.version=1.4
 commons-collections.version=3.1
 commons-httpclient.version=3.1
 commons-lang.version=2.5
-commons-logging.version=1.1.1
+commons-logging.version=1.1.3
 commons-logging-api.version=1.1
 commons-el.version=1.0
 commons-fileupload.version=1.2
-- 
1.7.9.5

